app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -318,12 +340,159 @@ describe('League entries page (Entry Component)', () => {
     await waitFor(() => {
       expect(screen.getByTestId('league-entry-logo')).toHaveAttribute(
         'src',
-        'team-a-logo.png',
+        '/_next/image?url=%2Fpackers-logo.png&w=96&q=75',
       );
 
       expect(screen.getByTestId('league-entry-pick-button')).toHaveTextContent(
         'Change Pick',
       );
     });
   });
+  it('should not display the user pick history if the current week is 1', async () => {
+    mockUseDataStore.mockReturnValue({
+      ...mockUseDataStore(),
+      currentWeek: 1,
+      NFLTeams: [
+        {
+          teamId: 'packers',
+          teamLogo: '/packers-logo.png',
+          teamName: 'Packers',
+        },
+      ],
+    });
+    mockGetCurrentUserEntries.mockResolvedValueOnce([
+      {
+        $id: '123',
+        name: 'Test Entry',
+        week: 1,
+        selectedTeams: ['Packers', 'Bears'],
+      },
+    ]);
+    mockGetGameWeek.mockResolvedValueOnce({ week: 1 });
+    mockGetCurrentLeague.mockResolvedValue({
+      leagueName: 'Test League',
+      participants: 10,
+      survivors: 8,
+    });
+    mockGetNFLTeams.mockResolvedValue([
+      { teamId: 'packers', teamLogo: '/packers-logo.png', teamName: 'Packers' },
+      { teamId: 'bears', teamLogo: '/bears-logo.png', teamName: 'Bears' },
+    ]);
+
+    render(<Entry params={{ leagueId: '123' }} />);
+
+    await waitFor(() => {
+      expect(mockGetGameWeek).toHaveBeenCalled();
+      expect(mockGetCurrentUserEntries).toHaveBeenCalled();
+      expect(mockGetCurrentLeague).toHaveBeenCalled();
+      expect(mockGetNFLTeams).toHaveBeenCalled();
+    });
+
+    const userPickHistory = screen.queryByTestId('user-pick-history');
+    expect(userPickHistory).not.toBeInTheDocument();
```

Comment:
For cleaner code, this could be one line

Reply:
✅ fixed!